Sources/XiEditor/XiViewProxy.swift

```diff
@@ -136,6 +145,32 @@ final class XiViewConnection: XiViewProxy {
         sendRpcAsync("find_next", params: params)
     }
 
+    /// transformations
```

Comment:
Would `// MARK: Tranformations` make sense here? Just recently came accross this here: https://stackoverflow.com/questions/35963128/swift-understanding-mark but I am not really a Swift developer.

Reply:
That's true! This will definitely improve the navigation 👍